index.html

```diff
@@ -423,6 +423,28 @@ <h2>
                       abort these steps. 
                     </p>
                   </li>
+                  <li>
+                    <p>Let <var>toplevelTraversable</var> be the <a>current settings object</a>'s's [=relevant global object=]'s
+                      [=navigable=]'s [=top-level traversable=].</p>
+                  </li>
+                  <li>
+                    <p>Listen to <var>toplevelTraversable</var>'s change of <a data-cite="!HTML/#system-focus">system focus</a>.</p>
+                    <p>The first time <var>toplevelTraversable</var> is losing <a data-cite="!HTML/#system-focus">focus</a>,
+                      queue a global task on the <a data-cite="!HTML/#user-interaction-task-source">user interaction task source</a>
```

Comment:
Mixing async instructions into sequential steps can be tricky to understand. At what point does this listening begin? i.e. what is it synced to? For how long should it listen? If I call getDisplayMedia() twice, are there two listeners? etc. There's also no such concept as being able to "listen" to changes like this.

In the end the user agent is listening to itself here, so it's not clear to me what the requirements actually are. Focus changes seem highly UA specific, so maybe it's better to just spell out the intent?

Rather than constrain the user agent, it sounds like what we're trying to describe here is actually a constraint on the feature. Might this be described as an after-thought to the previous step using something like: 

- Resolve p with stream.

   If a subsequent user action causes the _toplevelTraversable_ to lose [=system focus=] before a
   [focus decision has been finalized](https://w3c.github.io/mediacapture-screen-share/#dfn-finalize-focus-decision-algorithm), the User Agent MUST set controller</var>.[[FocusChangeDisabled]] to true.
- Abort the remaining steps.

Also, I don't think the User Agent should queue a task here, since the task to finalize the focus decision may already be in the queue, and if the user already switched focus, then we want the user to always win. AFAICT the [[FocusChangeDisabled]] internal slot isn't otherwise observable to JS, so this seems fine.

I also see we already have this (is it redundant now?):

_"The user agent MAY set [[[FocusChangeDisabled]]](https://w3c.github.io/mediacapture-screen-share/#dfn-focuschangedisabled) to true at any moment based on its own logic."_